__tests__/app.test.js

```diff
@@ -36,4 +37,31 @@ describe('api/topics',()=>{
         })
 
     })
+})
+
+describe('/api',()=>{
+    test('GET:200 responds with an object detailing all available endpoints in the app',()=>{
+        return request(app)
+        .get('/api')
+        .expect(200)
+         .then(({body:{endpoints}})=>{
+          
+          for(let endpoint in endpoints){
+            if(endpoint!="GET /api"){
+                console.log(endpoints[endpoint])
+                expect(endpoints[endpoint]).toEqual(
+                    expect.objectContaining({
```

Comment:
nice! We could go one step further and check that the values are also correct and make this test super dynamic by requiring the endpoints.json file into this test test file and asserting that the json response from the server is the same as the json from the file! :D